### DISCUSSION TITLE
Is my user variables configuration correct?

### DISCUSSION THREAD

**Discussion by hungnguyen1503** [dasupradyumna/launch.nvim#6] [Q&A]
Hi,

I'm using the launch plugins on Windows OS with this config. But it seems not to be correct. Could you please support me to fix it?
Thank you so much 💯 

``` lua
return {
    task = {
        {
            name = 'Test User Variables',
            command = 'echo',
            args = { '{@select_1}', '{@select_2}' },
        }
    },
    var = {
        select_1 = {
            type = 'select',
            desc = 'selection type variable',
            items = { 'ok', 'none ok', 'normal' }
        },
        select_2 = {
            type = 'select',
            desc = 'selection type variable',
            items = { '2', '3', '4' }
        },
    }
}
```
![image](https://github.com/dasupradyumna/launch.nvim/assets/58872428/829f5125-c33a-45d7-86a4-d8d9266e670a)

**Comment by dasupradyumna** [comment]
Does this not work on Ubuntu / Linux either?
I copied your exact config (on Ubuntu) and it worked as expected for me.

P.S. I do not have Windows system with me at the moment, but I will try it there too later.

**Comment by dasupradyumna** [comment]
Reopen if needed.
